{{cookiecutter.project_dashed}}-src/alembic/script.py.mako

```diff
@@ -7,18 +7,47 @@ Create Date: ${create_date}
 """
 from alembic import op
 import sqlalchemy as sa
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import Session
 ${imports if imports else ""}
 
 # revision identifiers, used by Alembic.
 revision = ${repr(up_revision)}
 down_revision = ${repr(down_revision)}
 branch_labels = ${repr(branch_labels)}
 depends_on = ${repr(depends_on)}
+metadata = sa.MetaData()
 
 
 def upgrade():
+    # run the data migration unless it is explicitly disabled - adjust as needed!
+    if context.get_x_argument(as_dictionary=True).get('data', 'true').lower() != 'false':
```

Comment:
What is this doing?  Why is it needed?

Reply:
This hooks into the alembic CLI to allow us to enable/disable data migrations when running schema updates. I have cleaned up imports/vars to segment those methods off if a dev wants to remove them outright from the migration. Also, the comment line has the CLI info needed for use.